### PR TITLE
Serving contracts using local file server

### LINKED ISSUE #5: Add scripts for deploying test project on development
https://github.com/FELToken/federated-learning-token/blob/main/scripts/deploy.py

### PR DESCRIPTION
**Pull request by Breta01**
Should solve issue #5 

- when run `brownie run deploy_dev` it starts http server at localhost:8100 serving the `build/deployments` folder 
- project contract is now copied to folder depending on network. So there aren't conflicts between dev and testnet.

### CODE REVIEW

**Review by FilipMasar — APPROVED** [review]
Update `README.md` pls

**Comment by FilipMasar** [comment]
I am also thinking what about cleaning `build/deployments` directory after I close server (stop brownie) .. because we don't want to push dev artifacts to github

**Comment by Breta01** [comment]
> about cleaning `build/deployments` directory after I close server (stop brownie) .. because we don't want to push dev artifacts to github

That automatically happens when you stop brownie command. You have to stop it using `Ctrl + C`

File: scripts/deploy_dev.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+# Total supply times decimals
+INITIAL_SUPPLY = 100000000000 * (10**18)
+
+
+def main():
+    owner = accounts.add(config["wallets"]["owner_key"])
+    print(f"On network {network.show_active()}")
+
+    # TODO: Provide intial supply to other accounts (node1, node2)
```
Comment (MartinOndejka) [inline scripts/deploy_dev.py:13]: Create issue pls for this and also creating sample project.

**Review by MartinOndejka — APPROVED** [review]
👍

**Pull request merged**